common/persistence/persistenceMetricClients.go

```diff
@@ -1463,6 +1463,7 @@ func (p *metricEmitter) updateErrorMetric(scope int, err error) {
 	case *serviceerror.NotFound, *serviceerror.NamespaceNotFound:
 		p.metricClient.IncCounter(scope, metrics.PersistenceErrEntityNotExistsCounter)
 	case *serviceerror.ResourceExhausted:
+		p.metricClient.Scope(scope, metrics.ResourceExhaustedCauseTag(err.Cause)).IncCounter(metrics.PersistenceErrBusyCounter)
 		p.metricClient.IncCounter(scope, metrics.PersistenceErrBusyCounter)
```

Comment:
Remove?

Reply:
will remove in separate PR along with bunch of other redundant metrics.